### PR TITLE
Issue91 enodeb class refactor

### PR DESCRIPTION
**Pull request by artuso0matteo**
This PR addresses issue #91 and includes the changes needed for the topology and creation of the network layout.
The scenario and test files have also been updated to use the new setup.

### CODE REVIEW

File: channel/tests/ChannelAPITest.m [hunk 1]
```diff
@@ -30,7 +30,8 @@ function createChannel(testCase)
 				testCase.Logger = MonsterLog(Config);
 				
 				Config.setupNetworkLayout(testCase.Logger);
-				Stations = setupStations(Config, testCase.Logger);
+				Sites = setupSites(Config, testCase.Logger);
+				Stations = [Sites.Cells];
```
Comment (jakthra) [inline channel/tests/ChannelAPITest.m]: The keyword stations should be removed and replaced with cells.

File: enb/EvolvedNodeB.m [hunk 2]
```diff
@@ -44,8 +45,13 @@
 	
 	methods
 		% Constructor
-		function obj = EvolvedNodeB(Config, BsClass, cellId, Logger)
+		function obj = EvolvedNodeB(Config, Logger, BsClass, position, cellId, antennaBearing, siteId, macroCellId)
```
Comment (jakthra) [inline enb/EvolvedNodeB.m]: Long function fingerprint. A solution has been discussed involving the use of the already created (and stored) network layout inside the Config object.

File: channel/tests/ChannelAPITest.m (hunk 1 above)
Comment (artuso0matteo) [reply channel/tests/ChannelAPITest.m]: Resolved in latest commit across project

File: enb/EvolvedNodeB.m (hunk 2 above)
Comment (artuso0matteo) [reply enb/EvolvedNodeB.m]: Resolved in latest commits with a site and cell configuration structure that is generated in the layout (for the sites) and in the site constructor (for the cells)

File: setup/MonsterConfig.m [hunk 3]
```diff
@@ -76,7 +76,7 @@
 			SimulationPlot.runtimePlot = 1;
 			SimulationPlot.generateCoverageMap = 0;
 			SimulationPlot.generateHeatMap = 0;
-			SimulationPlot.heatMapType = 'perStation';
+			SimulationPlot.heatMapType = 'perCell';
```
Comment (jakthra) [inline setup/MonsterConfig.m]: I think these parameters are unused. Same goes for coveragemap and heatmapres

**Pull request merged**